addons/membership/membership.py

```diff
@@ -488,6 +488,15 @@ def action_cancel(self, cr, uid, ids, context=None):
             member_line_obj.write(cr, uid, mlines, {'date_cancel': today})
         return super(Invoice, self).action_cancel(cr, uid, ids, context=context)
 
+    def unlink(self, cr, uid, ids, context=None):
+        member_line_obj = self.pool.get('membership.membership_line')
+        for invoice in self.browse(cr, uid, ids, context=context):
+            mlines = member_line_obj.search(cr, uid,
```

Comment:
You're right @pedrobaeza, but for upstream sometimes a simple 'space' at endline that dissapears in a PR is not good for Odoo (my text editor removes them automatically). 

In older PRs to Odoo I've learned that I have to do less changes as possible and follow its code rules in order to have more chances to be merged, so following that principle I try to follow its ways even if them are not good from OCA perspective. 

BTW It works, this is only a convention rule, isn't it?